projects/github.com/nishanths/license/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/nishanths/license/archive/refs/tags/v{{version}}.tar.gz
+
+warnings:
+  - vendored
+
+versions:
+  github: nishanths/license
+
+provides:
+  - bin/license
+
+build:
+  dependencies:
+    curl.se: '*'
+  env:
+    darwin/aarch64: {PLATFORM: darwin-arm64}
+    darwin/x86-64:  {PLATFORM: darwin-amd64}
+    linux/aarch64:  {PLATFORM: linux-arm64}
+    linux/x86-64:   {PLATFORM: linux-amd64}
+  working-directory: ${{prefix}}
+  script: |
+    curl -Lfo license "https://github.com/nishanths/license/releases/download/v{{version}}/license-v{{version}}-$PLATFORM"
+    mkdir bin
+    chmod +x license
+    mv license bin
```

Comment:
Where possible, we try to compile locally rather than use others' binary artifacts. Consider https://github.com/teaxyz/pantry.extra/blob/main/projects/xcfile.dev/package.yml as a minimal example of that.

Reply:
Okay I'll try my best to make it work that way over the weekend.